src/vision/frc_vision_final.py

```diff
@@ -28,19 +30,28 @@ def unequal(new, old_list):
     return True
 
 
+def newLine(filtered_lines, new_line, filtered_line_img, x1, y1, x2, y2):
+    filtered_lines.append(new_line)
```

Comment:
I don't think this will do anything if `filtered_lines` isn't global, which is something we typically try to avoid

Reply:
It will function fine - it'll operate on the parameter's reference, changing it.